src/app/infrastructure/app-routing.module.ts

```diff
@@ -58,6 +61,24 @@ const routes: Routes = [
     canActivate: [AuthGuard],
     data: { role: ['instructor'] },
   },
+  {
+    path: 'authoring/course/:courseId/unit/:unitId/learning-task/:ltId/domain-model',
```

Comment:
I think a better shorthand for `learning task` is `task` instead of `lt`. So it would be `taskId`.

Reply:
Ok